packages/teams-js/src/internal/utils.ts

```diff
@@ -39,6 +40,13 @@ function validateHostAgainstPattern(pattern: string, host: string): boolean {
 
 const validateOriginLogger = getLogger('validateOrigin');
 
+function validateOriginHandler(messageOrigin: URL): Promise<boolean> {
```

Comment:
>validateOriginHandler [](http://example.com/codeflow?start=9&length=21)

Given what it looks like this function is doing, consider a name that more clearly expresses the action it performs, e.g., `askHostIfDomainIsValid` or `checkIfDomainIsValid` or `validateDomainWithHost` or, my personal preference reflecting the `boolean` return value, `isDomainValidAccordingToHost`